template/build/webpack.prod.conf.js

```diff
@@ -14,6 +15,9 @@ var env = {{#if_or unit e2e}}process.env.NODE_ENV === 'testing'
   ? require('../config/test.env')
   : {{/if_or}}config.build.env
 
+var serviceWorkerLoader = `<script>${fs.readFileSync(path.join(__dirname,
```

Comment:
As this is only referenced once, shall we just inline it directly in `serviceWorkerLoader: <..>` lower down?